package.json

```diff
@@ -20,6 +20,9 @@
         "copy-scripts:array": "cp node_modules/posthog-js/dist/array.js* frontend/dist/",
         "copy-scripts:recorder": "cp node_modules/rrweb/dist/record/rrweb-record.min.js frontend/dist/recorder.js && cp node_modules/rrweb/dist/record/rrweb-record.min.js.map frontend/dist/recorder.js.map && cat node_modules/rrweb/dist/plugins/console-record.min.js >> frontend/dist/recorder.js && sed -i -e s/rrweb-record.min.js.map/recorder.js.map/ frontend/dist/recorder.js && sed -i -e 's/\\/\\/\\# sourceMappingURL=recorder/window.rrweb = {record: rrwebRecord}\\n\\/\\/# sourceMappingURL=recorder/g' frontend/dist/recorder.js",
         "test": "jest",
+        "test:sequential": "yarn test:chunk:1 && yarn test:chunk:2",
+        "test:chunk:1": "jest --testPathPattern='./frontend/src/scenes/.*'",
+        "test:chunk:2": "jest --testPathPattern='./frontend/src/(layout|lib|mocks|models|styles|test|toolbar)/.*'",
```

Comment:
can you do test path by exclusion instead?

So that one is "Everything in scenes" and the other "Everything not in scenes"

Saves someone adding a new folder and not running the tests in CI by accident

Reply:
Yep, https://jestjs.io/docs/configuration#testpathignorepatterns-arraystring